dll/win32/kernel32/kernel32_vista/CMakeLists.txt

```diff
@@ -13,7 +13,13 @@ list(APPEND SOURCE
     sync.c
     vista.c)
 
+if(DLL_EXPORT_VERSION GREATER_EQUAL 0x600)
+    list(APPEND SOURCE
+        ../client/console/vista.c)
+endif()
+
 add_library(kernel32_vista_static ${SOURCE})
+target_link_libraries(kernel32_vista_static ${PSEH_LIB})
```

Comment:
Shouldn't this be made conditional maybe also by 
if(DLL_EXPORT_VERSION GREATER_EQUAL 0x600)
?

Reply:
That doesn't change anything. And omitting that is shorter. And shorter is better, isn't it?

Reply:
Shorter **is** better if there would be no unintended change in behavior, yes.
I erroneously expected it to generate some linker warnings, same or similar as it would do for unused delayloads:
LINK : warning LNK4199: /DELAYLOAD:....dll ignored; no imports found from ....dll
But it seems it doesn't do any such checks for static imports.
I could not find any additional warning when I quickly checked the buildlogs of "gcc i386 dbg" and "MSVC i386 dbg" on the bots.